spec/models/product_importer_spec.rb

```diff
@@ -516,21 +524,27 @@
       expect(filter('invalid', entries)).to eq 2
 
       importer.entries.each do |entry|
-        expect(entry.errors.messages.values).to include ['cannot be updated on existing products via product import']
+        expect(entry.errors.messages.values)
+          .to include ['cannot be updated on existing products via product import']
       end
     end
   end
 
-  describe "when more than one product of the same name already exists with multiple variants each" do
+  describe "when more than one product of same name already exists with multiple variants each" do
     let(:csv_data) {
       CSV.generate do |csv|
         csv << ["name", "producer", "category", "description", "on_hand", "price", "units",
                 "unit_type", "display_name", "shipping_category"]
-        csv << ["Oats", enterprise.name, "Cereal", "", "50", "3.50", "500", "g", "Rolled Oats", shipping_category.name]   # Update
-        csv << ["Oats", enterprise.name, "Cereal", "", "80", "3.75", "500", "g", "Flaked Oats", shipping_category.name]   # Update
-        csv << ["Oats", enterprise.name, "Cereal", "", "60", "5.50", "500", "g", "Magic Oats", shipping_category.name]    # Add
-        csv << ["Oats", enterprise.name, "Cereal", "", "70", "8.50", "500", "g", "French Oats", shipping_category.name]   # Add
-        csv << ["Oats", enterprise.name, "Cereal", "", "70", "8.50", "500", "g", "Scottish Oats", shipping_category.name] # Add
+        csv << ["Oats", enterprise.name, "Cereal", "", "50", "3.50", "500",
+                "g", "Rolled Oats", shipping_category.name]   # Update
+        csv << ["Oats", enterprise.name, "Cereal", "", "80", "3.75", "500",
+                "g", "Flaked Oats", shipping_category.name]   # Update
+        csv << ["Oats", enterprise.name, "Cereal", "", "60", "5.50", "500",
+                "g", "Magic Oats", shipping_category.name]    # Add
+        csv << ["Oats", enterprise.name, "Cereal", "", "70", "8.50", "500",
+                "g", "French Oats", shipping_category.name]   # Add
+        csv << ["Oats", enterprise.name, "Cereal", "", "70", "8.50", "500",
+                "g", "Scottish Oats", shipping_category.name] # Add
```

Comment:
This is a lot harder to read. Maybe we should define the CSV as plain text, possibly with HEREDOC?